deployments/deploy-ssh.sh

```diff
@@ -75,6 +75,11 @@ if [ -z "$DRIVER_START_TIMEOUT" ]; then
     echo "Using default timeout of ${DRIVER_START_TIMEOUT}"
 fi
 
+APP_DESTROY_TIMEOUT_ARG=""
```

Comment:
Whats the default value for APP_DESTROY_TIMEOUT_ARG ?

Reply:
empty, it will be empty to take the default which is 5m

https://github.com/portworx/torpedo/blob/3fd89a2d1edebad1f8cc81fdc6a2b0cac00461f3/tests/common.go#L419

https://github.com/portworx/torpedo/blob/3fd89a2d1edebad1f8cc81fdc6a2b0cac00461f3/tests/common.go#L75